src/components/ha-icon-next.ts

```diff
@@ -6,7 +6,8 @@ import { HaIcon } from "./ha-icon";
 export class HaIconNext extends HaIcon {
   public connectedCallback() {
     this.icon =
-      window.getComputedStyle(this).direction === "ltr"
+      (document.querySelector("home-assistant")! as HTMLElement).style
```

Comment:
We cannot assume that this element is always available, like in Home Assistant Cast